plugin/src/test/java/org/opensearch/ml/utils/TestHelper.java

```diff
@@ -139,4 +150,30 @@ public static String httpEntityToString(HttpEntity entity) throws IOException {
         return sb.toString();
     }
 
+    public static RestRequest getRestRequest() {
```

Comment:
This method is to create REST request for KMeans, how about rename it as `getKMeansRestRequest` ?

Reply:
renamed them in the refreshment.